pkg/render/node.go

```diff
@@ -635,6 +635,12 @@ func (c *nodeComponent) cniEnvvars() []v1.EnvVar {
 			},
 		},
 	}
+
+	if c.cr.Spec.CalicoNetwork != nil {
```

Comment:
Should this be gated on Spec.Variant === Enterprise (like the others)?

Reply:
yes it should